src/ucp/proto/proto_debug.h

```diff
@@ -62,8 +53,8 @@ void ucp_proto_select_perf_str(const ucs_linear_func_t *perf, char *time_str,
                                size_t bw_str_max);
 
 
-void ucp_proto_select_init_trace_caps(const ucp_proto_init_params_t *init_params,
-                                      const ucp_proto_caps_t *proto_caps,
+void ucp_proto_select_init_trace_perf(const ucp_proto_init_params_t *init_params,
```

Comment:
`ucp_proto_debug_...` ?

Reply:
There are many functions in `proto_debug.h` which don't have `ucp_proto_debug_` prefix. Even function that was in that place before didn't have it. Why it should be applied here?

Reply:
I suggested this name, since you have changed it. So, I thought it would be better to start to use the proper naming.
However, I now have doubts about the name. On one hand, I would prefer to use the common prefix for the methods declared in the same file. On the other hand, there is not a single method named according to this pattern in this file.

Reply:
Let's keep the current names.